src/types/simulation.ts

```diff
@@ -169,8 +170,8 @@ export type SpanDB = {
   duration: string;
   durationMs: number;
   endMs: number;
-  id: SpanId;
   parent_id: number | null;
+  span_id: SpanId;
```

Comment:
Does this really need to be renamed to `span_id` from `id`? Isn't it already implied that the `id` on a `Span` is the same thing?